src/cgame/cg_event.cpp

```diff
@@ -102,8 +102,13 @@ static const struct {
 	{ "",              true,  false, TEAM_NONE },   // (MOD_REPLACE)
 };
 
+static Cvar::Cvar<bool> cg_showObituaries( "cg_showObituaries", "show obituaries in chat", Cvar::NONE, true );
```

Comment:
Nitpick: Maybe use a different wording like "death messages" in the description only as the word obituary may not be self-explanatory to non-english speakers.

Reply:
This problem would also apply to the variable's name. Not too fond of the wording "death messages" neither. And discovering cvars is hardly possible anyway... Either you know they exist, or you don't.

This whole obituaries thing needs proper work, and should not even be difficult to do, same as the whole chat situation, but this requires someone to tackle it for real. Which I don't think will be me, as I am barely able to add checkboxes without breaking formatting in rmlUI.

Reply:
My point was just to reduce redundancy in cvar name and description to make the latter potentially more helpful as an explanation. But this is a nitpick, so if you disagree then that resolves it.